### PR TITLE
Add inventory source detail view

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
Issue: #6574 

This PR adds the Inventory source details component and tests
* Delete button should delete the inventory source and its groups and hosts
  * Successful delete should navigate to inventory source list
  * Unsuccessful delete should show an alert modal and not navigate
* Edit button should navigate to the edit form (which is currently in a WIP) 

Screen Shot 2020-05-08 at 11 11 39 AM

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - UI

### CODE REVIEW

File: awx/ui_next/src/api/models/Inventories.js [hunk 1]
```diff
@@ -72,6 +72,22 @@ class Inventories extends InstanceGroupsMixin(Base) {
...
   }
+
+  async readSourceDetail(inventoryId, sourceId) {
+    const {
+      data: { results },
+    } = await this.http.get(
+      `${this.baseUrl}${inventoryId}/inventory_sources/?id=${sourceId}`
+    );
+
+    console.log(inventoryId, sourceId);
```
Comment (AlexSCorey) [inline awx/ui_next/src/api/models/Inventories.js]: remove this

**Review by AlexSCorey — APPROVED** [review]
You addressed all my concerns offline and I see you removed the errant console.log.  Good work!

File: awx/ui_next/src/api/models/Inventories.js (hunk 1 above)
Comment (marshmalien) [reply awx/ui_next/src/api/models/Inventories.js]: 😄"How did you get here?"

File: awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+              <CredentialChip key={cred?.id} credential={cred} isReadOnly />
+            ))}
+          />
+        )}
+        {source_regions && (
+          <Detail
+            fullWidth
+            label={i18n._(t`Regions`)}
+            value={
+              <ChipGroup numChips={5}>
```
Comment (nixocio) [inline awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx:160]: Perhaps, we could rebase this code, and use the new custom ChipGroup.

See: https://github.com/ansible/awx/pull/6935

**Pull request merged**